tests/queries/test_bulk_update.py

```diff
@@ -69,6 +71,15 @@ def test_batch_size(self):
         with self.assertNumQueries(len(self.notes)):
             Note.objects.bulk_update(self.notes, fields=["note"], batch_size=1)
 
+    def test_max_batch_size(self):
```

Comment:
This test and `bulk_create.tests.BulkCreateTests.test_max_batch_size` pass for me when I revert the changes 
Is this adding extra coverage (should probably pull to a separate commit) or do I need a specific SQLite version for these to fail?

Reply:
Thank you for the explanation
I think adding these 2 tests in a separate commit to add the extra coverage and then having the commit with the fix makes sense to me. If you can push that up, that would be super otherwise I can pick this up tomorrow :+1: